mqtt_client.c

```diff
@@ -990,6 +992,29 @@ static esp_err_t deliver_publish(esp_mqtt_client_handle_t client)
     return ESP_OK;
 }
 
+static esp_err_t deliver_suback(esp_mqtt_client_handle_t client)
+{
+    uint8_t *msg_buf = client->mqtt_state.in_buffer;
+    size_t msg_read_len = client->mqtt_state.in_buffer_read_len;
+    size_t msg_data_len = msg_read_len;
```

Comment:
`msg_read_len` is not used anywhere if I read correctly ->
```suggestion
    size_t msg_data_len = client->mqtt_state.in_buffer_read_len;
```

Reply:
I thought I had removed all unused. The compiler warnings disappeared.